service/geopmdpy/runtime.py

```diff
@@ -35,6 +35,8 @@
 
 """
 
+from __future__ import absolute_import
+from __future__ import division
```

Comment:
Similar issue with division, we are in the __future__ :)

Reply:
Sure, just trying to satisfy the test we have that checks all this stuff.  I can rip it all out in this PR too if that's the desire.